mico-core/src/main/java/io/github/ust/mico/core/dto/response/KFConnectorDeploymentInfoResponseDTO.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.github.ust.mico.core.dto.response;
+
+import io.github.ust.mico.core.dto.request.KFConnectorDeploymentInfoRequestDTO;
+import io.github.ust.mico.core.model.KFConnectorDeploymentInfo;
+import io.github.ust.mico.core.model.MicoServiceDeploymentInfo;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+
+/**
+ * DTO for {@link MicoServiceDeploymentInfo} intended to use with responses only.
+ */
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@Accessors(chain = true)
+public class KFConnectorDeploymentInfoResponseDTO extends KFConnectorDeploymentInfoRequestDTO {
```

Comment:
This class looks empty. Maybe just use one DTO for request and response as this is 54 lines of nonrelevant code.

Reply:
Currently it is empty, but if it exists it can be more easily extended without changing other code lines (if we would use the same DTO for both request and response).
That's why I'd like to keep it that way.

Reply:
Response DTO will be include the actual `KubernetesDeploymentInfo` in future. Will be covered by another issue (currently mentioned as a task in the epic #750)